### PR TITLE
Improvements

### PR DESCRIPTION
**Pull request by anandarh**
- Improve text style on covid-19 information screen
- Changed aspect ratio of the card items in the self-isolation guide list

### CODE REVIEW

File: lib/screens/home/components/VideoList.dart [hunk 1]
```diff
@@ -324,12 +324,14 @@ class _VideoListState extends State<VideoList> {
                                       margin: EdgeInsets.only(right: 5.0),
```
Comment (gaganugraha33) [inline lib/screens/home/components/VideoList.dart:324]: add `const` before `EdgeInsets` @anandarh

File: lib/screens/selfReport/EducationListScreen.dart [hunk 2]
```diff
@@ -145,77 +145,77 @@ class _EducationListScreenState extends State<EducationListScreen> {
...
+                          fontSize: 12.0,
+                          color: Colors.white,
+                          fontFamily: FontsFamily.roboto),
                     ),
-                  ),
-                ],
-              ),
-            )
-          ],
+                    SizedBox(
```
Comment (gaganugraha33) [inline lib/screens/selfReport/EducationListScreen.dart:198]: add `const` before `SizedBox` @anandarh

File: lib/screens/home/components/VideoList.dart (hunk 1 above)
Comment (anandarh) [reply lib/screens/home/components/VideoList.dart]: Done @gaganugraha33

File: lib/screens/selfReport/EducationListScreen.dart (hunk 2 above)
Comment (anandarh) [reply lib/screens/selfReport/EducationListScreen.dart:198]: Done @gaganugraha33

**Review by gaganugraha33 — APPROVED** [review]

**Review by jaluowen — APPROVED** [review]

**Pull request merged**